tests/integ/modin/frame/test_from_dict.py

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
If possible, can we support a dict where the values are Snowpark pandas series? I think that's the version that we need for compatibility with plotly>=6. e.g.

```python
 df = pd.DataFrame.from_dict({i: pd.Series(range(1)) for i in range(100)})
```

I tried this version on your branch, and it works, but also the number of queries scales with the size of the dict. In this example, I had to execute 200 non-describe queries and possibly more describe queries, and it looks like I materialized all the data at least once.

Reply:
I didn't think about that case - although I think it should be doable (mainly a concat of all the series).
I'd say create another ticket to have this additional support. And we can prioritize it later.

Reply:
Also, I think for plotting use cases, it's not very common to have too many series or series with very large data.

Reply:
We should add a test case for this use case, since we claim to support `from_dict` completely.

Reply:
Added a test.

Reply:
Should we add a TODO jira for supporting this use case lazily for performance improvement?

Reply:
Yeah, it's good to have a todo. Added.